### PR TITLE
CAPA: Release v30.0.0.

### PR DESCRIPTION
**Pull request by Gacko**
### Checklist

- [x] Roadmap issue created
- [x] Release uses latest stable Flatcar
- [x] Release uses latest Kubernetes patch version

### Triggering E2E tests

To trigger the E2E test for each new Release added in this PR, add a comment with the following:

`/run releases-test-suites PREVIOUS_RELEASE=29.1.0`

If you want to trigger conformance tests, you can do so by adding a comment similar to the following:

`/run conformance-tests PROVIDER=capa RELEASE_VERSION=30.0.0`

For more details see the [README.md](/README.md#running-tests-against-prs).

### CODE REVIEW

**Comment by Gacko** [comment]
Tests don't look good, gonna debug this later... 😔

File: capa/v30.0.0/release.yaml [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    catalog: cluster
+    version: 0.1.1
+    dependsOn:
+    - cilium
+  - name: node-exporter
+    version: 1.20.0
+    dependsOn:
+    - kyverno-crds
+  - name: observability-bundle
+    version: 1.6.1
```
Comment (TheoBrigitte) [inline capa/v30.0.0/release.yaml:92]:
```suggestion
  - name: observability-bundle
    version: 1.6.2
```

**Review by TheoBrigitte — COMMENTED** [review]
Can we add `observability-bundle` v1.7.0 within this release ?

**Comment by Gacko** [comment]
Sure, we can, but v30.0.0 is currently not work in progress.

**Comment by QuentinBisson** [comment]
There should also be a new capa release with auditd disabled IIUC. @giantswarm/team-tenet will we be able to add it here?

**Comment by Gacko** [comment]
Disabling auditd is a fix which needs to be back-ported to older releases of the `cluster` and `cluster-aws` chart. We are already handling this.

As stated before we currently have other priorities than releasing a v30.0.0. We first want to bring customers to v29.x.x. Therefore we'll soon come up with patched releases for releases from v25.0.0 and v29.0.0.

File: capa/v30.0.0/release.yaml [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+  - name: cert-manager
+    version: 3.8.1
+    dependsOn:
+    - prometheus-operator-crd
+  - name: chart-operator-extensions
+    version: 1.1.2
+    dependsOn:
+    - prometheus-operator-crd
+  - name: cilium
+    version: 0.27.0
```
Comment (ubergesundheit) [inline capa/v30.0.0/release.yaml:34]:
```suggestion
    version: 0.29.0
```

**Pull request closed without merging**